components/compliance-service/reporting/relaxting/suggestions.go

```diff
@@ -502,6 +502,170 @@ func (backend ES2Backend) getControlSuggestions(client *elastic.Client, typePara
 	return suggs, nil
 }
 
+func (backend ES2Backend) getControlTagsSuggestions(client *elastic.Client, typeParam string, target string, text string, controlTagFilterKey string, size int, filters map[string][]string, useSummaryIndex bool) ([]*reportingapi.Suggestion, error) {
+	if typeParam == "control_tag_value" && controlTagFilterKey == "" {
+		return nil, errors.New("'control_tag' filter is required for 'control_tag_value' suggestions")
```

Comment:
should this be using the "invalid" error option? i.e. https://github.com/chef/automate/blob/master/components/compliance-service/api/reporting/server/server.go#L78 ?
also maybe this should be at the api level since it's a validation path?

Reply:
I switched to invalid error, thank you.
We could validate the control_tag in the API, but will loop twice through the filters to extract the control tag key.